app/src/main/java/io/github/androidacademyglobal/profile/ProfileScreen.kt

```diff
@@ -0,0 +1,141 @@
+package io.github.androidacademyglobal.profile
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.ScrollState
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Divider
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.input.nestedscroll.nestedScroll
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import io.github.androidacademyglobal.R
+import io.github.androidacademyglobal.components.baselineHeight
+import io.github.androidacademyglobal.ui.theme.MARGIN_LARGE
+import io.github.androidacademyglobal.ui.theme.MARGIN_MEDIUM
+import io.github.androidacademyglobal.ui.theme.MARGIN_SMALL
+import io.github.androidacademyglobal.ui.theme.PROFILE_SCREEN_BOTTOM_MARGIN
+import io.github.androidacademyglobal.ui.theme.ZERO
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ProfileScreen(userData: ProfileScreenState) {
+
+    val scrollState = rememberScrollState()
+    val scrollBehavior = remember { TopAppBarDefaults.pinnedScrollBehavior() }
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .nestedScroll(scrollBehavior.nestedScrollConnection)
+    ) {
+        BoxWithConstraints(modifier = Modifier.weight(1f)) {
```

Comment:
How did you manage to leave these magic numbers and make detekt happy?)

Reply:
such numbers as +-1, +-2, +-0.5 should be added to detekt available consts